testsuite/tests/grafana/test_dashboards.py

```diff
@@ -0,0 +1,43 @@
+"""Tests for Grafana Dashboards definitions"""
+
+import json
+import pytest
+
+DASHBOARDS = [
+    ("apicast-mainapp",
+     "sum(rate(nginx_http_connections{namespace='$namespace', pod=~'apicast-$env.*'}[1m])) by (state)"),
+    ("apicast-services",
+     "sum(rate(upstream_status"
+     "{namespace='$namespace', pod=~'apicast-$env.*', service_id='$service_id'}[1m])) by (status)"),
+    ("backend",
+     "sum(rate(apisonator_listener_response_codes"
+     "{namespace=\\\"$namespace\\\",request_type=\\\"authorize\\\"}[1m]))"),
+    ("kubernetes-resources-by-namespace",
+     "sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate"
+     "{cluster=\\\"$cluster\\\", namespace=\\\"$namespace\\\"}) by (pod)"),
+    ("kubernetes-resources-by-pod",
+     "sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate"
+     "{namespace=\\\"$namespace\\\", pod=\\\"$pod\\\", container!=\\\"POD\\\", cluster=\\\"$cluster\\\"})"
+     " by (container)"),
+    ("system",
+     "sum(rate(rails_requests_total{namespace='$namespace',pod=~'system-app-[a-z0-9]+-[a-z0-9]+'}[1m])) by (status)"),
+    ("zync",
+     "sum(rate(rails_requests_total{namespace='$namespace',pod=~'zync-[a-z0-9]+-[a-z0-9]+'}[1m])) by (status)"),
+]
+
+
+@pytest.mark.smoke
+@pytest.mark.parametrize("dashboard,expresion", DASHBOARDS)
+def test_dashboards(openshift, dashboard, expresion):
+    """check expresion to be inside of json definition of the dashboard"""
+    client = openshift()
+    ret = client.do_action('get', ['GrafanaDashboard', dashboard, '-ojson'])
```

Comment:
`client.do_action('get', ['GrafanaDashboard', dashboard, '-ojson'], parse_output = True)` might solve the issue with big objects not being parsed as it returns APIObject, which can be used like `ret.model.spec.json` and shuld be already parsed